module/Activity/src/Activity/Service/ActivityCalendar.php

```diff
@@ -148,7 +148,13 @@ public function createProposal($data)
         $proposal->setName($name);
         $description = $validatedData['description'];
         $proposal->setDescription($description);
-        $proposal->setOrgan($this->sm->get('decision_service_organ')->getOrgan($organ));
+        if ($organ > -1) {
+            $proposal->setOrgan($this->sm->get('decision_service_organ')->getOrgan($organ));
+        } elseif ($organ == -1) {
+            $proposal->setOrganAlt("Board");
+        } elseif ($organ == -2) {
+            $proposal->setOrganAlt("Other");
+        }
```

Comment:
Please add a comment about the meaning or reason for the magic numbers -1 and -2. I checked and I think they're first defined in `module/Activity/src/Activity/Form/ActivityCalendarProposal.php` on line 33-34. Add this so future maintainers know where they appeared.

Reply:
Well, all organs have positive numbers for their IDs. I figured using some negative numbers would be useful for the non-organ options.